examples/tools/scene_viewer.rs

```diff
@@ -83,6 +84,27 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
     });
 }
 
+fn scene_drop(
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+    mut drop_events: EventReader<FileDragAndDrop>,
+) {
+    for event in drop_events.iter() {
+        if let FileDragAndDrop::DroppedFile { path_buf, .. } = event {
+            info!("Loading {}", path_buf.to_str().unwrap());
+
+            commands.insert_resource(SceneHandle {
```

Comment:
You can just use a ResMut here, which is clearer :)

I *think* the other incident of this can be changed to match; if you can get that to work please change it in this PR too.

Reply:
I could, but at least in theory this system should work regardless of whether that resource already exists, so I think this approach makes sense logically, and there's no reason to use Option<ResMut<_>> because we don't care whether or not it already exists or what's in it